src/compiler/interface2.jl

```diff
@@ -7,22 +7,24 @@ function edge!(m::IRTools.Meta, edge::Core.MethodInstance)
 end
 
 @generated function _pullback(ctx::AContext, f, args...)
-  T = Tuple{f,args...}
-  ignore_sig(T) && return :(f(args...), Pullback{$T}(()))
-
+  # Try using ChainRulesCore
   if is_kwfunc(f, args...)
     # if it is_kw then `args[1]` are the keyword args, `args[2]` is actual function
     cr_T = Tuple{ZygoteRuleConfig{ctx}, args[2:end]...}
     chain_rrule_f = :chain_rrule_kw
   else
     cr_T = Tuple{ZygoteRuleConfig{ctx}, f, args...}
+    Core.println("cr_T=", cr_T)
```

Comment:
Is this `println` meant for debugging?

Reply:
Other than this, LGTM.

Reply:
OOps i thought I deleted all of these,
Good catch
```suggestion
```